csvkit/utilities/csvtranspose.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+"""
+csvtranspose
+"""
+
+import agate
+
+from csvkit.cli import CSVKitUtility
+
+
+class CSVTranspose(CSVKitUtility):
+    description = 'Transpose CSV data.'
+
+    def add_arguments(self):
+        pass
+
+    def main(self):
+        self.args.columns = None
+        self.args.no_header_row = True
+
+        rows, column_names, column_ids = self.get_rows_and_column_names_and_column_ids(**self.reader_kwargs)
+        rows = map(list, zip(*rows))
+
+        column_names = list(rows[0])
```

Comment:
no: the contents of the first column of the original csv become the names of the columns in the transposed csv